tailwindcss/demo/tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./_site/**/*.{html,js}"],
```

Comment:
The content paths can be defined in the CLI with `--content="./_site/**/*.{html,js}"` (and should be configurable). This removes the requirement of creating a `tailwind.config.js` configuration file.